x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -41,7 +40,7 @@ export const getNetworkTopNFlowColumns = (
       const ipAttr = `${topNFlowType}.ip`;
       const ip: string | null = get(ipAttr, node);
       const id = escapeDataProviderId(
-        `networkTopNFlow-table-${uuid.v4()}-${topNFlowType}-${topNFlowDirection}-ip-${ip}`
+        `networkTopNFlow-table--${topNFlowType}-${topNFlowDirection}-ip-${ip}`
```

Comment:
Nice! Thanks! This looks to have fixed the re-registering of providers when re-rendering. 👍

Reply:
Thank you!!! For tracking this kind of behavior